New-Architecture/New-Architecture-in-depth.md

```diff
@@ -138,7 +138,7 @@ The **new Architecture** of **React Native** completes its full cycle in **2 pha
 
 ### Phase 1: App Build time
 
-When a developer turns on both **Codegen** and **Turbo Module** in their app and then gives a **build command** just before production deployment to create an **APK** for **Android** or an **IAP** for **iOS** device, the following steps are done from the new architecture flow:
+When a developer turns on both **Codegen** and **Turbo Module** in their app and then gives a **build command** just before production deployment to create an **APK** for **Android** or an **IPA** for **iOS** device, the following steps are done from the new architecture flow:
```

Comment:
Ohh yea, that should be **IPA** instead of **IAP**. Thanks. I will add here the full meaning of **IPA** too.